src/core/request.js

```diff
@@ -60,6 +60,8 @@ const createError = (response, fallbackMessage, httpCode, httpSent) => {
     let err;
     if (response && response.errorCode) {
         err = new Error(`${response.errorCode} : ${response.errorMsg || response.errorMessage || response.error}`);
```

Comment:
You can also keep the single message building considering the other variables 

```suggestion
    if (response && (response.errorCode || response.code)) {
        err = new Error(`${response.errorCode || response.code} : ${response.errorMsg || response.errorMessage || response.error || response.message}`);
```

Reply:
I wanted to seperate it because theres already lots of OR statements in this simple block of code and multiple ORs are harded to read.

Reply:
I understood you wanted to remove some cluttering, but it is also creating the impression we want different string formats inside the `new Error()`, which is not the case. So maybe you can separate these operations in variables for easy reading ?

either way, it is not a blocker :) 

Thanks!

Reply:
I've updated the code